app/src/main/java/com/architectcoders/arquitectomarvel/ui/main/AdapterList.kt

```diff
@@ -28,11 +29,11 @@ class AdapterList(private val listener: (Result, View) -> Unit) :
         val result = getItem(position)
         holder.bind(result)
         holder.itemView.setOnClickListener {
-            listener(result, it)
```

Comment:
@jujoru I did need to pass the view to animate the transition on MainActivity
```
ActivityOptionsCompat.makeSceneTransitionAnimation(this, view, getString(R.string.hero_image)
```
Why do you need to do that change?

Reply:
This is what I wanted to see with you on Wednesday because I did not understand well by receiving the sight, but now I see that it is for animation.
I write it down and adapt it.